docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -175,5 +173,53 @@ If a user were to scale the deployed example by patching the StatefulSet such th
 is fully shutdown and deleted. If web-0 were to fail after web-2 has been terminated and 
 is completely shutdown, but prior to web-1's termination, web-1 would not be terminated 
 until web-0 is Running and Ready.
+
+### Pod Management Policies
+In Kubernetes 1.7 and later, StatefulSet allows you to relax its ordering guarantees while 
+preserving its uniqueness and identity guarantees via its `.spec.podManagementPolicy` field.
+
+#### OrderedReady Pod Management
+
+`OrderedReady` pod management is the default for StatefulSets. It implements the behavior 
+described [above](#deployment-and-scaling-guarantees).
+
+#### Parallel Pod Management
+
+`Parallel` pod management tells the StatefulSet controller to launch or 
+terminate all Pods in parallel, and to not wait for Pods to becoming Running 
+and Ready or completely terminated prior to launching or terminating another 
+Pod.
+
+## Update Strategies
+
+In Kuberentes 1.7 and later, StatefulSet's `.spec.updateStrategy` field allows you to configure 
+and disable automated rolling updates for containers, labels, resource request/limits, and 
+annotations for the Pods in a StatefulSet.
+
+### On Delete
+
+The `OnDelete` update strategy implements the legacy (1.6 and prior) behavior. When a StatefulSet's 
+`.spec.updateStrategy.type` is set to `OnDelete`. The StatefulSet controller will not automatically 
+update the Pods in a StatefulSet. Users must manually delete Pods to cause the controller to 
+create new Pods that reflect modifications made to a StatefulSet's `.spec.template`.
+
+### Rolling Updates
+
+The `RollingUpdate` update strategy implements automated, rolling update for the Pods in a 
+StatefulSet. When a StatefulSet's `.spec.updateStrategy.type` is set to `RollingUpdate`, the 
+StatefulSet controller will delete and recreate each Pod in the StatefulSet. It will proceed 
+in the same order as Pod termination, updating each Pod one at a time. It will wait until an 
+updated Pod is Running and Ready prior to updating its predecessor.
+
+#### Partitions
```

Comment:
This needs an explanation of why you would want to set this field.
Something like:

> In most cases, you'll want to leave use `.spec.updateStrategy.rollinUpdate.partition` unset.   Use it when you need to pause the rollout at a specific index, and leave it at that state for a while (such as to observe stability of the system, run tests, etc).